libdvdcss2.mk

```diff
@@ -0,0 +1,47 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+SUBPROJECTS        += libdvdcss2
+LIBDVDCSS2_VERSION := 1.4.2
+DEB_LIBDVDCSS2_V   ?= $(LIBDVDCSS2_VERSION)
+
+libdvdcss2-setup: setup
+	wget -q -nc -O$(BUILD_SOURCE)/libdvdcss2-$(LIBDVDCSS2_VERSION).tar.bz2  https://download.videolan.org/pub/libdvdcss/$(LIBDVDCSS2_VERSION)/libdvdcss-$(LIBDVDCSS2_VERSION).tar.bz2
+	$(call EXTRACT_TAR,libdvdcss2-$(LIBDVDCSS2_VERSION).tar.bz2,libdvdcss-$(LIBDVDCSS2_VERSION),libdvdcss2)
+
+ifneq ($(wildcard $(BUILD_WORK)/libdvdcss2/.build_complete),)
+libdvdcss2:
+	@echo "Using previously built libdvdcss2."
+else
+libdvdcss2: libdvdcss2-setup
+	cd $(BUILD_WORK)/libdvdcss2 && ./configure -C \
+		$(DEFAULT_CONFIGURE_FLAGS)
+	+$(MAKE) -C $(BUILD_WORK)/libdvdcss2
+	+$(MAKE) -C $(BUILD_WORK)/libdvdcss2 install \
+		DESTDIR=$(BUILD_STAGE)/libdvdcss2
+	touch $(BUILD_WORK)/libdvdcss2/.build_complete
+endif
+
+libdvdcss2-package: libdvdcss2-stage
+	# libdvdcss2.mk Package Structure
+	mkdir -p $(BUILD_DIST)/libdvdcss{2,-dev}/usr/lib
+	
+	# libdvdcss2.mk Prep libdvdcss2
+	cp -a $(BUILD_STAGE)/libdvdcss2/usr/lib/*.dylib $(BUILD_DIST)/libdvdcss2/usr/lib
```

Comment:
Use `$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)`
and don't wildcard copy this. there's a symlink from the lib to the soversioned lib. This is put inside of `libdvdcss-dev`